lib/hanami/utils/string.rb

```diff
@@ -151,6 +152,10 @@ def classify
         self.class.new words.zip(delimiters).join
       end
 
+      def classified?
+        split(NAMESPACE_SEPARATOR).reduce(false) {|memo, str| str.match(/\A[A-Z][a-zA-Z_]*/).to_s.length == str&.length }
```

Comment:
I think it's so complicated method and we need to simplify it.

That's why I suggest:
1. Set regexp to constant
2. Instead `#reduce` you can use [`#all?`](https://apidock.com/ruby/Enumerable/all%3F) method
3. I think you can use `/\w+/` instead `/[a-zA-Z_]/`